src/main/java/be/seeseemelk/mockbukkit/potion/MockInternalPotionData.java

```diff
@@ -104,23 +103,32 @@ public int getMaxLevel()
 		return maxLevel;
 	}
 
-	private List<PotionEffect> getPotionEffectFromData(JsonObject data)
+	// TODO probably not the right solution to make this public static, but where should this utility method be located?
```

Comment:
I think it's a necessary evil to accomplish what you're trying to do here. With the above suggestion I believe it's fine